platform/darwin/src/MGLBackgroundStyleLayer.mm

```diff
@@ -84,13 +85,33 @@ - (void)removeFromMapView:(MGLMapView *)mapView
 
 #pragma mark - Accessing the Paint Attributes
 
+- (NSArray *)transitionKeys
```

Comment:
Ah, looking more closely, this method appears to be used only in tests (but is exposed publicly). As an alternative to this method, could we move the array of keys into the tests themselves?